### PR TITLE
[KYUUBI #2087] Add issue template for documentation improvement

### PR DESCRIPTION
**Pull request by yangrong688**
### _Why are the changes needed?_

close https://github.com/apache/incubator-kyuubi/issues/2087
Add an issue template for documentation improvement.
I have self-tested this doc issue template and shown as below.

image

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/doc-improvement-report.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Kyuubi Doc Improvement Report
+title: "[DOCS] "
+description: Problems and issues with docs of Apache Kyuubi
+labels: ["kind:bug", "kind:documentation"]
```
Comment (yaooqinn) [inline .github/ISSUE_TEMPLATE/doc-improvement-report.yml:21]: A doc Improvement may not be kind of bug

File: .github/ISSUE_TEMPLATE/doc-improvement-report.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Kyuubi Doc Improvement Report
+title: "[DOCS] "
+description: Problems and issues with docs of Apache Kyuubi
```
Comment (yaooqinn) [inline .github/ISSUE_TEMPLATE/doc-improvement-report.yml:20]: Fix errors, or improve the content or refactor architecture of online documentation

File: .github/ISSUE_TEMPLATE/doc-improvement-report.yml (hunk 1 above)
Comment (yangrong688) [reply .github/ISSUE_TEMPLATE/doc-improvement-report.yml]: I have removed label "kind:bug".

File: .github/ISSUE_TEMPLATE/doc-improvement-report.yml (hunk 2 above)
Comment (yangrong688) [reply .github/ISSUE_TEMPLATE/doc-improvement-report.yml]: I have updated description of this issue template.

**Comment by yaooqinn** [comment]
thanks, @yangrong688
You may also update other places based on my suggestions, such as `What do you see as an issue?` as it's not an issue.

**Comment by yangrong688** [comment]
> thanks, @yangrong688 You may also update other places based on my suggestions, such as `What do you see as an issue?` as it's not an issue.

Hi @yaooqinn , I have updated descriptions and placeholders based on your suggestion.
Shall I also add version info of kyuubi in this issue template?

**Comment by yaooqinn** [comment]
> Shall I also add version info of kyuubi in this issue template?

SGTM

**Comment by yangrong688** [comment]
> > Shall I also add version info of kyuubi in this issue template?
> 
> SGTM

Done, please help to review if you have time.

**Review by yaooqinn — APPROVED** [review]

**Pull request closed without merging**